The `WKInterfaceImage+Kingfisher.swift` should be exclused for the macos executable project by use SPM?

i build a executable project by swiftPM, and add the dependencies like this

```swift
import PackageDescription

let package = Package(
    name: "FeedbackBuilder",
    dependencies: [
        // Dependencies declare other packages that this package depends on.
        // .package(url: /* package url */, from: "1.0.0"),
        .package(url: "https://github.com/Moya/Moya.git", .upToNextMajor(from: "11.0.0")),
        .package(url: "https://github.com/onevcat/Kingfisher.git", .upToNextMajor(from: "4.9.0")),
        .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", .upToNextMinor(from: "0.9.0")),
        .package(url: "https://github.com/SwiftyJSON/SwiftyJSON.git", .upToNextMinor(from: "4.0.0")),
        .package(url: "https://github.com/stephencelis/SQLite.swift.git", .upToNextMinor(from: "0.11.5")),
    ],
    targets: [
        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
        .target( 
            name: "FeedbackBuilder",
            dependencies: ["RxMoya", "Kingfisher", "CryptoSwift", "SwiftyJSON", "SQLite"]),
    ]
)

```

but when i build , it's wrong.

```bash
Updating https://github.com/krzyzanowskim/CryptoSwift.git
Updating https://github.com/Moya/Moya.git
Updating https://github.com/Quick/Nimble.git
Updating https://github.com/ReactiveX/RxSwift.git
Updating https://github.com/ReactiveCocoa/ReactiveSwift.git
Updating https://github.com/Alamofire/Alamofire.git
Updating https://github.com/SwiftyJSON/SwiftyJSON.git
Updating https://github.com/antitypical/Result.git
Updating https://github.com/stephencelis/SQLite.swift.git
Updating https://github.com/Quick/Quick.git
Updating https://github.com/onevcat/Kingfisher.git
Resolving https://github.com/onevcat/Kingfisher.git at 4.8.1
Compile SQLiteObjc SQLite-Bridging.m
Compile Swift Module 'SwiftyJSON' (1 sources)
Compile Swift Module 'Result' (2 sources)
Compile Swift Module 'RxSwift' (149 sources)
Compile Swift Module 'Alamofire' (17 sources)
Compile Swift Module 'Kingfisher' (23 sources)
Compile Swift Module 'CryptoSwift' (67 sources)
Compile Swift Module 'SQLite' (22 sources)
/Users/three/Desktop/GitLib/FeedbackBuilder/.build/checkouts/Kingfisher.git-1970406525084737423/Sources/WKInterfaceImage+Kingfisher.swift:27:8: error: no such module 'WatchKit'
import WatchKit
       ^
/Users/three/Desktop/GitLib/FeedbackBuilder/.build/checkouts/Kingfisher.git-1970406525084737423/Sources/WKInterfaceImage+Kingfisher.swift:27:8: error: no such module 'WatchKit'
import WatchKit
       ^
/Users/three/Desktop/GitLib/FeedbackBuilder/.build/checkouts/Kingfisher.git-1970406525084737423/Sources/WKInterfaceImage+Kingfisher.swift:27:8: error: no such module 'WatchKit'
import WatchKit
       ^
/Users/three/Desktop/GitLib/FeedbackBuilder/.build/checkouts/Kingfisher.git-1970406525084737423/Sources/WKInterfaceImage+Kingfisher.swift:27:8: error: no such module 'WatchKit'
import WatchKit
       ^
error: terminated(1): /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-build-tool -f /Users/three/Desktop/GitLib/FeedbackBuilder/.build/debug.yaml main output:
```

is the `WKInterfaceImage+Kingfisher.swift` should be exclused for the macos executable project?

--- a/Package.swift
+++ b/Package.swift
@@ -28,5 +28,5 @@ import PackageDescription
 
 let package = Package(
   name: "Kingfisher",
-  exclude: ["Tests","Sources/AnimatedImageView.swift","Sources/UIButton+Kingfisher.swift"]
+  exclude: ["Tests", "Sources/AnimatedImageView.swift", "Sources/UIButton+Kingfisher.swift", "Sources/WKInterfaceImage+Kingfisher.swift"]
 )

